### PR TITLE
Fixing elasticsearch v2 test

### PR DESCRIPTION
**Pull request by roysagi**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/20334

## Description
Disabled warnings for insecure connection

### CODE REVIEW

File: Integrations/Elasticsearch_v2/Elasticsearch_v2.py [hunk 1]
```diff
@@ -10,9 +10,11 @@
 from datetime import datetime
 import json
 import requests
+import warnings
```
Comment (ShahafBenYakir) [inline Integrations/Elasticsearch_v2/Elasticsearch_v2.py]: Is this part of default python? Or you needed to install it?

Comment (roysagi) [reply Integrations/Elasticsearch_v2/Elasticsearch_v2.py]: didn't need to install it.

Comment (ShahafBenYakir) [reply Integrations/Elasticsearch_v2/Elasticsearch_v2.py]: Sounds good!

**Comment by roysagi** [comment]
adding docs approved as there is no change in Changelog/yml/description.

**Pull request merged**